cmd/kubectl-direct_csi/drives_accesstier_set.go

```diff
@@ -108,7 +106,7 @@ func setAccessTier(ctx context.Context, accessTier directcsi.AccessTier) error {
 			setDriveAccessTier(drive, accessTier)
 			return nil
 		},
-		defaultDriveUpdateFunc(directCSIClient),
+		defaultDriveUpdateFunc(client.GetUnversionedDirectCSIDriveClientset()),
```

Comment:
You could name it as `client.GetDirectCSIClientset()` instead of `client.GetUnversionedDirectCSIClientset()`. The specific version of clientset is created by importing specific version so that this name does not conflict.

Reply:
Renamed it to `GetDirectCSIDriveClientset`